templates/solaris_start_puppet.sh.erb

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+puppet_pid="<%= @puppet_agent_pid %>"
```

Comment:
An improvement was made to the Windows script to pass in the PID (so the template isn't regenerated on each run). I'll do that here.